content/en/platform/corda/4.10/community/_index.md

```diff
@@ -1,23 +1,26 @@
 ---
-date: '2020-04-07T12:00:00Z'
+date: '2023-02-28'
 menu:
   versions:
     weight: -650
   corda-community-4-10:
+    identifier: about-corda-landing-4-10-community
     weight: -5
-    name: Corda Community Edition 4.10
+    name: Corda Community 4.10
 project: corda
 section_menu: corda-community-4-10
-title: Corda Community Edition 4.10
+title: Corda Community 4.10
 version: 'Community 4.10'
 ---
 
-# Welcome to Corda Community Edition 4.10
+# Corda Community Edition 4.10
 
 Corda Community Edition is a free to use, open source version of Corda, with added optional support. By choosing to develop using this edition of Corda, you can access affordable technical support to help you take your project to market with confidence.
 
 {{< vimeo 205410473 >}}
 
+Corda is the world’s first private, permissioned distributed ledger technology (DLT) platform designed to work with today’s financial services industry. While regulated companies may start on public blockchains, they soon realize when they get to production, that they require capabilities native to Corda such as privacy, security, scalability, and ease-of-integration with existing systems. That’s why R3 is also exploring interoperability with assets that originate on a non-Corda network.
+
 ## What's new in Corda Community Edition
```

Comment:
Was expecting Release Notes based on the heading.

Reply:
I removed the heading entirely as it doesn't seem to be required.